CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.10)
-project(credentials-fetcher)
+project(credentials-fetcher VERSION 1.2.0)
```

Comment:
I am okay with this, but, this version will need to be kept same as https://github.com/aws/credentials-fetcher/blob/mainline/package/credentials-fetcher.spec#L66.
Optionally, you can add a comment.

Reply:
ok